sp800-63b/sec4_aal.md

```diff
@@ -130,8 +130,9 @@ Authentication Assurance Level 3 requires the use of one of three kinds of hardw
  * Single-Factor Cryptographic Device
 * Multi-Factor Cryptographic Device
 * Single-Factor Cryptographic Device used in conjunction with Memorized Secret
+* Multi-Factor OTP Device used in conjunction with a single-factor cryptographic device
```

Comment:
Maybe I wasn't following the Ray thread but MFA in conjunct with SF crypto looks like 2-haves and 1-know.  What am I missing?